src/ci/github-actions/jobs.yml

```diff
@@ -542,13 +542,13 @@ auto:
     env:
       RUST_CONFIGURE_ARGS: --build=x86_64-pc-windows-msvc --enable-sanitizers --enable-profiler
       SCRIPT: make ci-msvc-py
-    <<: *job-windows-25
+    <<: *job-windows
```

Comment:
Suggestion: can you also add a FIXME or NOTE about why we're using windows-2022 images?

Reply:
added a note 👍